R/driver-oracle.R

```diff
@@ -122,7 +122,12 @@ setMethod("odbcConnectionColumns_", c("Oracle", "character"),
       )
     }
 
-    dbGetQuery(conn, query)
+    res <- dbGetQuery(conn, query)
+
+    res$data_type <- as.numeric(res$data_type)
```

Comment:
Do we need to revert this column back to another type after processing it?

Reply:
Hey Simon:

Good question - I think the query can/does return all `NULL`s for that field, which in turn I think gives a column full of `NA_character_` in the data-frame.